### PR TITLE
Nats streaming sample with Golang

### PR DESCRIPTION
**Pull request by balchua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,5 +24,6 @@
 * [RabbitMQ consumed by Go container](https://github.com/kedacore/sample-go-rabbitmq)
 * [RabbitMQ consumed by .NET container](https://github.com/NileshGule/pd-tech-fest-2019#prerequisites)
 
-
+#### Nats Streaming
```
Comment (tomkerkhove) [inline README.md]: Nitpick - Can we alphabetically order it please?

**Review by tomkerkhove — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (balchua) [reply README.md]: No problem.

**Comment by tomkerkhove** [comment]
Paging @jeffhollan & @ahmelsayed to have a look

**Comment by jeffhollan** [comment]
Thanks for submitting!

**Pull request merged**